core/arch/arch.c

```diff
@@ -741,8 +741,9 @@ d_r_arch_init(void)
            offsetof(app_state_at_intercept_t, start_pc) + sizeof(void *));
     /* Try to catch errors in x86.asm offsets for dcontext_t */
```

Comment:
Add `XXX i#7226: Use asm_offsetsx.h for systematic offset checks like on aarchxx.`

Reply:
Done.